graphql/types/OrderForm.graphql

```diff
@@ -55,6 +55,13 @@ type OrderFormItem {
   isGift: Boolean
   parentItemIndex: Int
   parentAssemblyBinding: String
+  assemblyOptions: AssemblyOptionItem
+}
+
```

Comment:
We need to document better. Can you add docs for each type and your variables? 

This will be exposed in graphiql docs tool.

Reply:
will do.